vocabulary/modified.csv

```diff
@@ -0,0 +1,10 @@
+field_id,name,mapping_approach,date_field_id,modified_by_field_id,modified_by_value,target_concept_id,target_value_concept_id,target_unit_concept_id
+3894,Age heart attack diagnosed,Modified,53,6150,1,40770406,4329847,9448
+3627,Age angina diagnosed,Modified,53,6150,2,40770406,77670,9448
+4056,Age stroke diagnosed,Modified,53,6150,3,40770406,4310996,9448
+2966,Age high blood pressure diagnosed,Modified,53,6150,4,40770406,4086587,9448
+20551,Substance of prescription or over-the-counter medication addiction,Modified,53,20503,1,3184281,4156741,
+20551,Substance of prescription or over-the-counter medication addiction,Modified,53,20503,1,3184281,4189272,
+20551,Substance of prescription or over-the-counter medication addiction,Modified,53,20503,1,3184281,45878142,
+2986,Started insulin within one year diagnosis of diabetes,Modified,53,31/2443/4041,1/1,4188893,45884660,
+2986,Started insulin within one year diagnosis of diabetes,Modified,53,31/2443/4041,0/1/0,4188893,45884660,
```

Comment:
How to interpret the 31/2443/4041 value for `modified_by_field_id`? Might be a good example for the lookup script.